tests/FSharp.Compiler.ComponentTests/EmittedIL/GenericComparison/Compare02.fsx.il.bsl

```diff
@@ -70,50 +70,58 @@
       IL_0003:  nop
       IL_0004:  ldc.i4.0
       IL_0005:  stloc.1
-      IL_0006:  br.s       IL_0027
+      IL_0006:  br.s       IL_0030
 
       IL_0008:  ldc.i4.1
       IL_0009:  ldc.i4.1
       IL_000a:  cgt
-      IL_000c:  stloc.2
-      IL_000d:  ldloc.2
-      IL_000e:  brfalse.s  IL_0014
-
-      IL_0010:  ldloc.2
-      IL_0011:  nop
-      IL_0012:  br.s       IL_0022
-
-      IL_0014:  ldc.i4.2
-      IL_0015:  ldc.i4.2
-      IL_0016:  cgt
-      IL_0018:  stloc.3
-      IL_0019:  ldloc.3
-      IL_001a:  brfalse.s  IL_0020
-
-      IL_001c:  ldloc.3
-      IL_001d:  nop
-      IL_001e:  br.s       IL_0022
-
-      IL_0020:  ldc.i4.m1
+      IL_000c:  ldc.i4.0
+      IL_000d:  sub
```

Comment:
The subtraction of zero again has be a little perplexed.  Any idea what's going on here?